pkg/ovs/ovs-vsctl.go

```diff
@@ -126,7 +133,10 @@ func SetInterfaceBandwidth(iface, ingress, egress string) error {
 		}
 		if egressBPS > 0 {
 			if len(qosList) == 0 {
-				qos, err := ovsCreate("qos", "type=linux-htb", fmt.Sprintf("other-config:max-rate=%d", egressBPS), fmt.Sprintf("external-ids:iface-id=%s", iface))
+				qos, err := ovsCreate("qos", "type=linux-htb",
+					fmt.Sprintf("other-config:max-rate=%d", egressBPS),
+					fmt.Sprintf("external-ids:iface-id=%s", iface),
+					fmt.Sprintf("external-ids:pod=%s/%s", podNamespace, podName))
```

Comment:
We should check empty `podName` or `podNamespace` here. `podName` and `podNamespace` passed by `func (c *Controller) setGatewayBandwidth()` are empty.

Reply:
> We should check empty `podName` or `podNamespace` here. `podName` and `podNamespace` passed by `func (c *Controller) setGatewayBandwidth()` are empty.

It's been fixed now.